package.json

```diff
@@ -16,10 +16,11 @@
     "build": "cross-env NODE_ENV=production npm run build:svelte && npm run build:electron",
     "build:svelte": "vite build",
     "build:electron": "electron-builder --x64 --win --config build.config.json ",
-    "build:mac-arm": "cross-env NODE_ENV=production vite build && electron-builder --mac --arm64 --config build.config.json",
-    "build:mac-intel": "cross-env NODE_ENV=production vite build && electron-builder --mac --x64 --config build.config.json",
-    "publish:mac-arm": "cross-env NODE_ENV=production vite build && electron-builder --mac --arm64 --config build.config.json -p always",
-    "publish:mac-intel": "cross-env NODE_ENV=production vite build && electron-builder --mac --x64 --config build.config.json -p always",
+    "build:mac-arm64": "cross-env NODE_ENV=production vite build && electron-builder --mac --arm64 --config build.config.json",
```

Comment:
Changing these names to be more clear what target instead of intel since we can have both 32 and 64bit builds. Even if that will not happen, it's good to have clarity.